jepsen/src/jepsen/nemesis/time.clj

```diff
@@ -138,35 +138,50 @@
     (teardown! [_ test]
       (reset-time! test))))
 
-(defn reset-gen
-  "Randomized reset generator. Performs resets on random subsets of the tests'
+(defn reset-gen-select
```

Comment:
These aren't generators any more; they're functions which return generators, and their use pattern is going to be different. Might want to make that explicit in the docstrings.

Reply:
Fixed.